.gitignore

```diff
@@ -42,4 +42,4 @@ docs
 .idea
 
 # sqlite db
-*.db.*
+cell-dev.sqlite
```

Comment:
Isn't dev db stored under `userData` as well that we don't generate it inside project folder and need to ignore it?

Reply:
It's a problem that when I use `typeorm` `cli` to generate migration, it only follows config in `ormconfig.json`, if no `database` here, it will throw error. So it's only use is `cli` need one ..

The real database in our project is under `userData`, it set dynamically in code while initialize connection (should get `userData` dir firstly)

Reply:
Got it. We should always put DB and other store files under `userData` for all environment, but if it's a limitation of `typeorm cli` for local operation then it's fine.